src/cljam/algo/pileup.clj

```diff
@@ -158,23 +310,15 @@
      :or {min-base-quality 13 min-map-quality 0 ignore-overlaps? false}}]
    (when-let [len (:len (refs/ref-by-name (sam/read-refs sam-reader) chr))]
      (let [s (max 1 start)
-           e (min len end)]
-       (->> {:chr chr :start s :end e}
-            (sam/read-alignments sam-reader)
-            (sequence
-             (comp
-              (filter (basic-mpileup-pred min-map-quality))
-              (map index-cigar)))
-            (pileup-seq s e)
-            (sequence
-             (comp (map resolve-bases)
-                   (if ignore-overlaps?
-                     identity
-                     (map correct-overlaps))
```

Comment:
Please delete unused private var `correct-overlaps` from `src/cljam/algo/pileup.clj` and `test/cljam/algo/pileup_test.clj`

Reply:
Thanks!
Removed them.